utils/coreutils/cmdutils.go

```diff
@@ -158,6 +159,17 @@ func ExtractDetailedSummaryFromArgs(args []string) (cleanArgs []string, detailed
 	return
 }
 
+func ExtractXrayScanFromArgs(args []string) (cleanArgs []string, xrayScan bool, err error) {
```

Comment:
The ```ExtractXrayScanFromArgs```,  ```ExtractDetailedSummaryFromArgs``` and ```ExtractInsecureTlsFromArgs``` functions all have some common code that can be taken out into a new provate function.